.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -0,0 +1,102 @@
+name: Bug Report
+description: Use this template reporting bugs and unexpected behavior.
+labels: [bug, needs triage]
+body:
+  - type: markdown
+    attributes:
+      value: "# Bug Report"
+  
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for opening a bug report! Please take the time to properly and thoughtfully fill
+        out this form to make it easier for us to work with it
+
+        If you notice you cant fully express your issue in this template, you can always open a
+        blank template with all the creative freedom youll need
```

Comment:
There should be a link to the blank template here!

Reply:
I dont think you can link to that sort of thing since its a dialog, not a page

Reply:
damn